src/test/daml/Daml/Finance/Account/Test/Controllers.daml

```diff
@@ -45,16 +54,23 @@ ownerControlled = script do
   TestInitialState {issuerAccount; investorAccount; issuerHoldingCid} <-
     setupInitialState tp (Fungible.Factory with provider = custodian; observers = pp) [] Owner
 
+  -- Build index table
+  idx <- Holding.buildIndexTableForTransfer [custodian] issuerHoldingCid investorAccount
+
   -- Custodian can't transfer.
   submitMultiMustFail [custodian] [publicParty] do
     exerciseCmd (coerceInterfaceContractId @Transferable.I issuerHoldingCid)
-      Transferable.Transfer with actors = S.fromList [custodian]; newOwnerAccount = investorAccount
+      Transferable.Transfer with
+        actors = S.fromList [custodian]; newOwnerAccount = investorAccount; idx
+
+  -- Build index table
+  idx <- Holding.buildIndexTableForTransfer [issuer, investor] issuerHoldingCid investorAccount
```

Comment:
no need to re-create table

Reply:
this is done to check proper contract visibility